src/resources/breaking-changes.md

```diff
@@ -0,0 +1,828 @@
+---
+title: Breaking changes
+description: A list of breaking changes by release in Dart.
+---
+
+This page lists the breaking changes in Dart, organized by release and area.
+Complete release notes are available in the [Dart SDK change log][change log].
+
+According to our [breaking change policy], each listed change contains a link to more
+information on the change, such as migration guidance, background, reasoning,
+caveats, and more.
+
+{{site.alert.note}}
+Changes prior to Dart 3.1.0 and our updated breaking change policy 
+might not have links to additional information. More details on some of those
+changes might be available in their corresponding [change log] entry.
```

Comment:
Maybe we can just keep this for a while. I think most users will be looking for new releases rather those old ones, so this note will slow down the average use case.

Reply:
I agree. How long do you think makes sense? I could also just not include it at all, open to either